Sophos/sophos firewall/ingest/parser.yml

```
@@ -61,6 +64,8 @@ stages:
           source.port: "{{kv_event.message.srcport or kv_event.message.src_port}}"
           destination.port: "{{kv_event.message.dstport or kv_event.message.dst_port or kv_event.message.port}}"
 
+          network.protocol: "{{kv_event.message.caller}}"
```

[`network.application`](https://www.elastic.co/guide/en/ecs/8.17/ecs-network.html#field-network-application) is more suitable:
```suggestion
          network.application: "{{kv_event.message.caller}}"
```